### ISSUE TITLE
[Bug] Missing hash key: "text" (KeyError)

### ISSUE THREAD

**Issue by MINETUBE02** [iv-org/invidious#3096]
**Describe the bug**

**Steps to Reproduce**

1. Go to 'https://yewtu.be/watch?v=3vaLxo-79nM'.
2. See error.

**Logs**

Title: `Missing hash key: "text" (KeyError)`
Date: `2022-05-12T15:16:27Z`
Route: `/watch?v=3vaLxo-79nM`
Version: `2022.05.12-123d619 @ master`

<details>
<summary>Backtrace</summary>
<p>
   
```
Missing hash key: "text" (KeyError)
  from /usr/lib/crystal/core/json/any.cr:100 in '[]'
  from /usr/lib/crystal/core/json/any.cr:215:3 in 'content_to_comment_html'
  from src/invidious/videos.cr:1057:27 in 'extract_video_info:video_id'
  from /usr/lib/crystal/core/hash.cr:1185:13 in 'fetch_video'
  from src/invidious/videos.cr:1126:13 in 'get_video:region'
  from src/invidious/routes/watch.cr:63:15 in 'handle'
  from lib/kemal/src/kemal/route.cr:13:9 in '->'
  from lib/kemal/src/kemal/config.cr:92:7 in 'call'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/filter_handler.cr:21:7 in 'call'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call_next'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call_next'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call_next'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call_next'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call_next'
  from src/ext/kemal_static_file_handler.cr:167:15 in 'call'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call'
  from /usr/lib/crystal/core/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/init_handler.cr:12:7 in 'process'
  from /usr/lib/crystal/core/http/server.cr:500:5 in '->'
  from /usr/lib/crystal/core/fiber.cr:146:11 in 'run'
  from /src/boringssl/src/google-boringssl-251b516/ssl/internal.h:334:8 in '??'
```
</p>
</details>

**Screenshots**

![Bug Screenshot](https://user-images.githubusercontent.com/55697597/168113890-5df41fbf-2670-4bd8-85ed-1cbcf8f1549a.png)

**Additional context**

- Browser Chrome
- OS Windows 10 Pro x64
- Version 20H2
- OS Build 19042.985

**Comment by SamantazFox** [comment]
Thanks for reporting this! seems like youtube is now returning empty description elements? Wtf....

```json
"description": {
  "runs": [
    {
      "text": "Here is an amazing....."
    },
    {},   <- HERE
    {
      "text": "... WinRAR: "
    },
    [SNIP]
```

**Comment by TheFrenchGhosty** [comment]
Fixed in #3099
